lib/requestDecorator.js

```diff
@@ -0,0 +1,502 @@
+/*!
+ * express
+ * Copyright(c) 2009-2013 TJ Holowaychuk
+ * Copyright(c) 2013 Roman Shtylman
+ * Copyright(c) 2014-2015 Douglas Christopher Wilson

```

Comment:
@dougwilson can you confirm this copyright header is fine?

Reply:
If it correctly represents the copyrights of the contents that were moved then it's fine. I can dig in sometime after the holiday if it is an open question.